packages/lesswrong/lib/collections/tags/schema.ts

```diff
@@ -618,7 +618,6 @@ const schema: SchemaType<DbTag> = {
     canCreate: ['admins'],
     group: formGroups.advancedOptions,
     nullable: true,
-    ...schemaDefaultValue(""),
```

Comment:
I removed the default values for these two fields because I think this is what was preventing me from submitting the tag edit form to clear them out, and they don't need default values anyway